### DISCUSSION TITLE
How to solve Apache rewrite error?

### DISCUSSION THREAD

**Discussion by geestream** [datenstrom/community#297] [Ask a question] [answered]
Hi, I just uploaded Yellow to my server but I'm getting [this error](http://guillaumeb.com/journal): _"Something went wrong. Rewrite module not enabled on this server!"_. I tried with PHP 4.4, 5.6 and 7.0

My website is hosted at OVH and they apparently did activate the rewrite module. I added a .htaccess file with this in it : 
```
<ifModule mod_rewrite.c>
RewriteEngine On
</ifModule>
```
But it does not seem to do the job either...

**Comment by schulle4u** [comment, marked as answer]
You will need to add some rewrite rules for Yellow. Try this .htaccess file which also can be found in every Yellow release. 

```
<IfModule mod_rewrite.c>
RewriteEngine on

DirectoryIndex index.html yellow.php
RewriteRule ^(cache|content|system)/ error [L]
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule ^ yellow.php [L]
</IfModule>
```

HTH
Steffen

**Comment by geestream** [comment]
thanks, it worked :) 
</Gee <http://guillaumeb.com/>>
